### PR TITLE
[pt] Updating Ecto/Querying lesson in Portuguese

### PR DESCRIPTION
**Pull request by elainenaomi**
Hi all 👋  

Following the instruction from [Issue #2395](https://github.com/elixirschool/elixirschool/issues/2395), I updated the Ecto Querying lesson in Portuguese.

Please, let me know if there're expressions or words that we normally don't translate. 
I added a suggestion with the English term for some cases (e.g. for bindings and bindingless).

Thank you all for your contribution and effort to maintain these lessons 💜

### CODE REVIEW

File: pt/lessons/ecto/querying.md [hunk 1]
```diff
@@ -90,122 +88,99 @@ iex> Repo.all(query)
...
-Observe que essa abordagem ao uso de `select` retorna uma lista contendo os valores selecionados.
+É sempre uma boa ideia começar com uma simples consulta sem atribuição (bindingless) e introduzir a atribuição sempre que você precisar referenciar sua estrutura de dados. Mais sobre atribuições (bindings) em consultas pode ser encontrado na [documentação do Ecto](https://hexdocs.pm/ecto/Ecto.Query.html#module-query-expressions)
+
+### Consultas baseadas em macros
 
-### Usando Expressões `where`
+Nos exemplos acima nós usamos as palavras-chave `select:` e `where:` dentro da macro `from` para construir uma consulta (query) - essas são chamadas de *consultas baseadas em palavras-chave*. Há, no entanto, outra forma de compor consultas - baseadas em macros. Ecto fornece macros para cada palavra-chave, como `select/3` or `where/3`.
 
-Podemos usar expressões `where` para incluir cláusulas `where` em nossas consultas. Várias expressões `where` são combinadas em instruções SQL `WHERE AND`.
+Cada macro aceita um valor *queryable* (buscável), *uma lista explícita de bindings* e a mesma expressnao que você forneceu para sua consulta de palavras-chave análoga:
```
Comment (anderkonzen) [inline pt/lessons/ecto/querying.md:153]: Just a small typo here: "... a mesma expressnao que ..."

**Review by anderkonzen — APPROVED** [review]
Hi @elainenaomi ! Thanks for this update! ❤️ 
I just found two small typos on the new translations.

File: pt/lessons/ecto/querying.md [hunk 2]
```diff
@@ -90,122 +88,99 @@ iex> Repo.all(query)
...
-
-* Uma expressão `in` (ex: `m in Movie`)
-* Um módulo que implementa o protoculo `Ecto.Queryable` (ex: `Movie`)
-
-O segundo argumento é nossa consulta de palavra-chave `select` .
-
-#### Usando `from` com uma Query Expression
-
-Ao usar `from` com uma expressão de consulta, o primeiro argumento deve ser um valor que implemente o protocolo `Ecto.Queryable` (ex: `Movie`). O segundo argumento é uma expressão. Vamos ver um exemplo: 
+O exemplo acima não tem a parte mais divertida das declarações (statements) SQL. Nós frequentemente queremos não apenas buscar campos específicos or filtrar registros por alguma condição. Vamos carregar `title` e `tagline` de todos os filmes que tenham o título `"Ready Player One"`:
```
Comment (elainenaomi) [inline pt/lessons/ecto/querying.md:93]:
```suggestion
O exemplo acima não tem a parte mais divertida das declarações (statements) SQL. Nós frequentemente queremos não apenas buscar campos específicos ou filtrar registros por alguma condição. Vamos carregar `title` e `tagline` de todos os filmes que tenham o título `"Ready Player One"`:
```

File: pt/lessons/ecto/querying.md [hunk 3]
```diff
@@ -90,122 +88,99 @@ iex> Repo.all(query)
...
-Observe que essa abordagem ao uso de `select` retorna uma lista contendo os valores selecionados.
+É sempre uma boa ideia começar com uma simples consulta sem atribuição (bindingless) e introduzir a atribuição sempre que você precisar referenciar sua estrutura de dados. Mais sobre atribuições (bindings) em consultas pode ser encontrado na [documentação do Ecto](https://hexdocs.pm/ecto/Ecto.Query.html#module-query-expressions)
+
+### Consultas baseadas em macros
 
-### Usando Expressões `where`
+Nos exemplos acima nós usamos as palavras-chave `select:` e `where:` dentro da macro `from` para construir uma consulta (query) - essas são chamadas de *consultas baseadas em palavras-chave*. Há, no entanto, outra forma de compor consultas - baseadas em macros. Ecto fornece macros para cada palavra-chave, como `select/3` or `where/3`.
 
-Podemos usar expressões `where` para incluir cláusulas `where` em nossas consultas. Várias expressões `where` são combinadas em instruções SQL `WHERE AND`.
+Cada macro aceita um valor *queryable* (buscável), *uma lista explícita de bindings* e a mesma expressnao que você forneceu para sua consulta de palavras-chave análoga:
```
Comment (elainenaomi) [inline pt/lessons/ecto/querying.md:153]:
```suggestion
Cada macro aceita um valor *queryable* (buscável), *uma lista explícita de bindings* e a mesma expressão que você forneceu para sua consulta de palavras-chave análoga:
```

File: pt/lessons/ecto/querying.md (hunk 1 above)
Comment (elainenaomi) [reply pt/lessons/ecto/querying.md]: Nice catch! Thanks!

File: pt/lessons/ecto/querying.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (elainenaomi) [reply pt/lessons/ecto/querying.md]: I didn't notice 😱 
Thanks!

**Review by anderkonzen — APPROVED** [review]

**Pull request merged**